common/src/main/java/io/honeycomb/opentelemetry/EnvironmentConfiguration.java

```diff
@@ -147,6 +154,12 @@ public static boolean isPresent(String value) {
         return value != null && !value.isEmpty();
     }
 
+    public static boolean isLegacyKey(String key) {
+        // legacy key has 32 characters
+        // todo: check multi-line input
+        return key.matches("^[0-9a-fA-F]{32}$");
```

Comment:
Like we discussed out-of-band, we might dial this down to a simple length check.